File: JCZhang-Taller/JCZhang-Taller-Java/src/main/java/models/Electricista.java
package models;



public class Electricista extends Trabajador implements iElectricista {

    String nombre;
    int aniosExp;
    int horasDiarias;

    public Electricista(String nombre, int aniosExp, int horasDiarias) {
        super(nombre, aniosExp, horasDiarias);
        this.nombre = nombre;
        this.aniosExp = aniosExp;
        this.horasDiarias = horasDiarias;
    }


    @Override
    public void comer() {
        System.out.println("Electricista: Hola! soy " +nombre+ " y estoy comiendo");

    }

    @Override
    public void arreglarElectricidad(){
        System.out.println("Electricista: Estoy arreglando los circuitos eléctricos");
    }

    @Override
    public String toString() {
        return "Electricista (" +nombre+" con " + aniosExp +"años de experiencia trabaja "+ horasDiarias+ " al día)";

    }

}

File: JCZhang-Taller/JCZhang-Taller-Kotlin/src/main/kotlin/models/Electricista.kt
package models

class Electricista(nombre: String, aniosExp: Int, horasDiarias: Int) : Trabajador(nombre, aniosExp, horasDiarias), iElectricista{

    fun Electricista(nombre: String, aniosExp: Int, horasDiaras: Int) {
        this.nombre = nombre
        this.aniosExp = aniosExp
        this.horasDiarias = horasDiaras
    }

    override fun comer() {
        println("Electricista: Hola! soy $nombre y estoy comiendo")
    }

    override fun arreglarElectricidad(){
        println("Electricista: Estoy arreglando los circuitos eléctricos")
    }

    override fun toString(): String {
        return "Electricista ($nombre con $aniosExp años de experiencia trabaja $horasDiarias horas al dia)"
    }

}